tutorials/lodash/capitalize-first-letter.md

```diff
@@ -0,0 +1,21 @@
+If you want to capitalize the first letter of every word in a string, you can use lodash's `capitalize()` function.
+
+```javascript
+const _ = require('lodash');
+
+_.capitalize('hello world, today is a beautiful day'); // Hello World, Today Is A Beautiful Day
```

Comment:
Did you test this? According to my tests and [the lodash docs](https://lodash.com/docs/#capitalize):

```
> const lo = require('lodash')
undefined
> lo.capitalize('hello world, today is a beautiful day')
'Hello world, today is a beautiful day'
> 
```

Reply:
I don't know why I thought capitalize did that

Reply:
Don't take my word for it or the docs' word for it. Always try it out yourself.